src/Portal.js

```diff
@@ -11,6 +11,9 @@ import ownerDocument from './utils/ownerDocument';
  * You can think of it as a declarative `appendChild()`, or jQuery's `$.fn.appendTo()`.
  * The children of `<Portal/>` component will be appended to the `container` specified.
  */
+
+const useCreatePortal = ReactDOM.createPortal !== undefined;
```

Comment:
Should we also support the deprecated `ReactDOM.unstable_ createPortal`? IIRC, it's only present in React 16 pre-releases.

Reply:
i'd go with "no"; just support the releases i think